e2e/customized-env/customized_env_suite_test.go

```diff
@@ -41,6 +41,7 @@ var _ = BeforeSuite(func() {
 	appName = fmt.Sprintf("e2e-customizedenv-%d", time.Now().Unix())
 	vpcConfig = client.EnvInitRequestVPCConfig{
 		CIDR:               "10.1.0.0/16",
+		AZs:                `""`, // Use default AZ options.
```

Comment:
Why does it have to be `""` instead of an empty string 🤔

Reply:
I wasn't sure how the `os/exec` pkg would behave if the argument is the empty string, I think it will do `--override-az-names --override-public-cidrs` instead of `--override-az-names "" --override-public-cidrs` that's why I explicitly marked it